.devcontainer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM fedora:latest
 RUN dnf install -y \
     bat \
+    erofs-latest \
```

Comment:
The container image used for testing is actually defined in [osbuild/containers](https://github.com/osbuild/containers). So I think that you need to add `erofs-latest` here: https://github.com/osbuild/containers/blob/13c129e9d51b4c1094df8ccfad71f176b77910a3/docker-bake.hcl#L231

btw, I'm very happy to discuss if we should move the testing container image into osbuild/osbuild itself. I find this setup confusing. (note that the setup IIRC predates GHCR, so that should explain why it feels weird to a 2023 reader.).

Reply:
If we could do that I would like to move the container here and make it a slightly simpler setup, a single `Containerfile` should suffice.